### PR TITLE
refactor(*): de-angular-ize task reader/writer/executor

### PR DESCRIPTION
**Pull request by anotherchrisberry**
Also taskMonitor

### CODE REVIEW

**Comment by jrsquared** [comment]
lets find out

File: app/scripts/modules/core/src/task/monitor/taskMonitorService.js [hunk 1]
```diff
@@ -1,86 +0,0 @@
...
-        monitor.submitting = true;
-        monitor.task = null;
-        monitor.error = false;
-        monitor.errorMessage = null;
-      };
-
-      monitor.setError = function(task) {
-        if (task) {
-          monitor.task = task;
-          monitor.errorMessage = monitor.task.failureMessage || 'There was an unknown server error.';
```
Comment (christopherthielen) [inline app/scripts/modules/core/src/task/monitor/taskMonitorService.js]: @anotherchrisberry where did all this code go?

Comment (anotherchrisberry) [reply app/scripts/modules/core/src/task/monitor/taskMonitorService.js]: Dead code - it hasn't been used anywhere since...early last year, I think?

**Comment by anotherchrisberry** [comment]
If I were a betting man I would say this is fine.

**Pull request merged**